packages/k4gen/package.py

```diff
@@ -9,7 +9,8 @@ class K4gen(CMakePackage, Key4hepPackage):
 
     maintainers = ['vvolkl']
 
-    version('master', branch='master')
+    version('main', branch='main')
+    version('master', branch='main')
```

Comment:
I don't think the version('master' ... ) line is needed - I only did this once in a package which had an existing installation with the master version to avoid an error,  but I doubt this applies here.

Reply:
Indeed I don't think it applies, just in case it's somewhere as `k4gen@master` but probably not, at least not in the github repos

Reply:
This is removed now